lib/helpers.bash

```diff
@@ -149,26 +149,70 @@ _bash-it-plugins ()
     _bash-it-describe "plugins" "a" "plugin" "Plugin"
 }
 
-_bash-it_update() {
+_bash-it_update-dev() {
+  _about 'updates Bash-it to the latest master'
+  _group 'lib'
+
+  _bash-it_update- dev "$@"
+}
+
+_bash-it_update-stable() {
+  _about 'updates Bash-it to the latest tag'
+  _group 'lib'
+
+  _bash-it_update- stable "$@"
+}
+
+_bash-it_pull_and_update_inner() {
+  git checkout "$1" &> /dev/null
+  if [[ $? -eq 0 ]]; then
+    echo "Bash-it successfully updated."
+    echo ""
+    echo "Migrating your installation to the latest $2 version now..."
+    _bash-it-migrate
+    echo ""
+    echo "All done, enjoy!"
+    bash-it reload
+  else
+    echo "Error updating Bash-it, please, check if your Bash-it installation folder (${BASH_IT}) is clean."
+  fi
+}
+
+_bash-it_update-() {
   _about 'updates Bash-it'
+  _param '1: What kind of update to do (stable|dev)'
   _group 'lib'
 
+  declare silent
+  for word in $@; do
+    if [[ ${word} == "--silent" || ${word} == "-s" ]]; then
+      silent=true
+    fi
+  done
   local old_pwd="${PWD}"
 
   cd "${BASH_IT}" || return
 
-  if [ -z $BASH_IT_REMOTE ]; then
+  if [ -z "$BASH_IT_REMOTE" ]; then
     BASH_IT_REMOTE="origin"
   fi
+  # Defaults to stable update
+  if [ -z "$1" ] || [ "$1" == "stable" ]; then
+    version="stable"
+    TARGET=$(git describe --tags "$(git rev-list --tags --max-count=1)")
```

Comment:
With the current lack of tags, this is causing the following error to be printed to `stderr`(?):

```bash
❯ git describe --tags "$(git rev-list --tags --max-count=1)"
fatal: No names found, cannot describe anything.
```

We should probably hide this error, i.e. pipe it to `/dev/null`, like the other `git` commands further down.

Reply:
done, and also added a check for this case, so a nice massage will be shown